src/components/post/PostField/PostField.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { PostField as PostFieldComponent } from './index'
+
+type PostField = typeof PostFieldComponent
+
+const meta: Meta<PostField> = {
+  component: PostFieldComponent,
+  title: 'Product/PostField'
```

Comment:
```suggestion
  title: 'Post/PostField'
```
전체적으로 product -> post로 변경된게 맞을까요??

Reply:
넵 맞습니다 ! be쪽에서 상품상세api param을 postId로 사용하고 있어서 혼란이 없도록 product -> post로 변경했습니다.

Reply:
[반영 완료](https://github.com/price-offer/offer-fe/pull/187/commits/760b33cd8bd4e6b8e89aa4720aa7302736d82878) !